quicknote/popup/quicknote.css

```diff
@@ -0,0 +1,123 @@
+/* General styling */
+
+* {
+  box-sizing: border-box;
+}
+
+html,body {
+  margin: 0;
+}
+
+html {
+  font-family: sans-serif;
+  font-size: 10px;
+  background: background: rgb(240,240,240);
+  height: 300px;
+}
+
+body {
+  width: 300px;
+  background: rgb(240,240,240);
+  margin: 0 auto;
```

Comment:
I'm not sure why you are defining margin at line 8 and again here. Why not move the declaration at 8 into the rule for html? Is there a thing I'm missing here?